packages/venia-concept/server-lambda.js

```diff
@@ -0,0 +1,42 @@
+process.chdir(__dirname);
+const validEnv = require('./validate-environment')(process.env);
+const {
+    bestPractices,
+    createUpwardServer,
+    envToConfig
+} = require('@magento/upward-js');
+
+async function serve() {
+    const config = Object.assign({}, envToConfig(validEnv), {
+        env: validEnv,
+        before: app => {
+            app.use(bestPractices());
+        },
+        bindLocal: false,
+        logUrl: false
```

Comment:
`bindLocal` and `logUrl` can be and are set from the environment variables passed in the `now.json` file. You could remove lines 15 and 16 I think.